src/pages/profile/index.js

```diff
@@ -5,19 +5,22 @@ export const Profile = () => {
   const profileBody = document.createElement('div');
   const profileHeader = document.createElement('header');
   const profileBodyNewReview = document.createElement('div');
+  const user = firebase.auth().currentUser;
   profileBodyNewReview.className = 'review-container';
   profileHeader.className = 'profile-header';
+  console.log(user);
 
   profileHeader.innerHTML = `
     <div id="profile-banner">IMG-BANNER</div>
     <div class="profile-user-info">
-      <h2 id="profile-user-name">Samara Tokki</h2>
+      <h2 id="profile-user-name">${user.displayName}</h2>
+      <img class="profile-user-avatar" src="${user.photoURL || '../../img/default_user_icon.jpg'}">
```

Comment:
O usuário de novo cadastro já é criado com essa imagem padrão, então não precisa chamar de novo aqui...

```suggestion
      <img class="profile-user-avatar" src="${user.photoURL}">
```